cloudbuild.yaml

```diff
@@ -1,16 +1,16 @@
 steps:
```

Comment:
Could you give a rough overview of what the build is doing at the top of the file. The line comments indicate how it's doing things, but don't necessarily give the bigger picture.